src/Alpacka.Lib/Pack/Config/EntryDefaults.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using YamlDotNet.Core;
+using YamlDotNet.Serialization;
+
+namespace Alpacka.Lib.Pack.Config
+{
+    public class EntryDefaults : ICollection<EntryDefaults.Group>
+    {
+        public EntryDefaults()
```

Comment:
so why is that stuff hardcoded and there is this same info in the default.yaml ? is this supposed to be all implied ?